shell/platform/embedder/embedder.h

```diff
@@ -31,6 +31,119 @@ typedef enum {
+  kFlutterAccessibilityFeatureAccessibleNavigation = 1 << 0,
+  // Indicate the platform is inverting the colors of the application.
+  kFlutterAccessibilityFeatureInvertColors = 1 << 1,
+  // Request that animations be disabled or simplified.
+  kFlutterAccessibilityFeatureDisableAnimations = 1 << 2,
+  // Request that text be rendered at a bold font weight.
+  kFlutterAccessibilityFeatureBoldText = 1 << 3,
+  // Request that certain animations be simplified and parallax effects
+  // removed.
+  kFlutterAccessibilityFeatureReduceMotion = 1 << 4,
+} FlutterAccessibilityFeature;
+
+// The set of possible actions that can be conveyed to a semantics node.
+typedef enum {
+  // The equivalent of a user briefly tapping the screen with the finger without
+  // moving it.
+  kFlutterSemanticsActionTap = 1 << 0,
+  // The equivalent of a user pressing and holding the screen with the finger
+  // for a few seconds without moving it.
+  kFlutterSemanticsActionLongPress = 1 << 1,
+  // The equivalent of a user moving their finger across the screen from right
+  // to left.
+  kFlutterSemanticsActionScrollLeft = 1 << 2,
+  // The equivalent of a user moving their finger across the screen from left to
+  // right.
+  kFlutterSemanticsActionScrollRight = 1 << 3,
+  // The equivalent of a user moving their finger across the screen from bottom
+  // to top.
+  kFlutterSemanticsActionScrollUp = 1 << 4,
+  // The equivalent of a user moving their finger across the screen from top to
+  // bottom.
+  kFlutterSemanticsActionScrollDown = 1 << 5,
+  // Increase the value represented by the semantics node.
+  kFlutterSemanticsActionIncrease = 1 << 6,
+  // Decrease the value represented by the semantics node.
+  kFlutterSemanticsActionDecrease = 1 << 7,
+  // A request to fully show the semantics node on screen.
+  kFlutterSemanticsActionShowOnScreen = 1 << 8,
+  // Move the cursor forward by one character.
+  kFlutterSemanticsActionMoveCursorForwardByCharacter = 1 << 9,
+  // Move the cursor backward by one character.
+  kFlutterSemanticsActionMoveCursorBackwardByCharacter = 1 << 10,
+  // Set the text selection to the given range.
+  kFlutterSemanticsActionSetSelection = 1 << 11,
+  // Copy the current selection to the clipboard.
+  kFlutterSemanticsActionCopy = 1 << 12,
+  // Cut the current selection and place it in the clipboard.
+  kFlutterSemanticsActionCut = 1 << 13,
+  // Paste the current content of the clipboard.
+  kFlutterSemanticsActionPaste = 1 << 14,
+  // Indicate that the node has gained accessibility focus.
+  kFlutterSemanticsActionDidGainAccessibilityFocus = 1 << 15,
+  // Indicate that the node has lost accessibility focus.
+  kFlutterSemanticsActionDidLoseAccessibilityFocus = 1 << 16,
+  // Indicate that the user has invoked a custom accessibility action.
+  kFlutterSemanticsActionCustomAction = 1 << 17,
+  // A request that the node should be dismissed.
+  kFlutterSemanticsActionDismiss = 1 << 18,
+} FlutterSemanticsAction;
+
+typedef enum {
```

Comment:
Can there be some reuse with https://github.com/flutter/engine/blob/6c68bf3caf89ea3cf9a17e52437a9937054e25a2/lib/ui/semantics/semantics_node.h#L49?

Reply:
That is not C though.